spec/spec.ddoc

```diff
@@ -13,6 +13,13 @@ IMPLEMENTATION_DEFINED=$(DIVC spec-boxes implementation-defined, $(B Implementat
 
 $(COMMENT Identifies undefined-defined behavior in the spec)
 UNDEFINED_BEHAVIOR=$(DIVC spec-boxes undefined-behavior, $(B Undefined Behavior): $0)
+
+$(COMMENT While not part of the spec, an explanation for something unexpected)
+RATIONALE= $(DIVC spec-boxes rationale, $(B RationaleBest Practices:) $0)
```

Comment:
missing space there? between Rationale and Best or just delete a word

Reply:
Copy pasta error alert!

Reply:
fixed